lib/CPAN/Mini/Inject.pm

```diff
@@ -420,7 +425,19 @@ sub inject {
      or croak "Copy $source to $tdir failed: $!";
 
     $self->_updperms( $target );
-    print "$target ... injected $module\n" if $verbose;
+  }
+
+  # if verbose report target file and the modules it contains
+  if ( $verbose ) {
+    for my $target (keys %report) {
```

Comment:
Maybe the keys should be sorted? That would make reading the report easier. Since we cannot have them in the order they were done, we can at least have them in some order, rather than random.